backend/schema/schema_test.go

```diff
@@ -387,7 +387,7 @@ func TestParseModule(t *testing.T) {
 module todo {
   config configValue String
   secret secretValue String
-  database testdb
+  postgres database testdb
```

Comment:
nice

Reply:
I like it, but `database` should come first `database <type> <name>`